test/selenium/OneOffContributionsSpec.scala

```diff
@@ -20,7 +20,7 @@ class OneOffContributionsSpec extends FeatureSpec with GivenWhenThen with Before
 
   feature("Sign up for a one-off contribution") {
 
-    scenario("One-off contribution sign-up with Stripe - GBP") {
+    scenario("One-off contribution sign-up with Stripe - AUD") {
```

Comment:
We are happy to leave this one in are we?

Reply:
This one should be very reliable (famous last words), as it directly calls the callback rather than interacting with the Stripe UI: https://github.com/guardian/support-frontend/blob/09610f52ec1f416137a30dab23d6dd545d6906ec/assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.jsx#L73-L74